tests/resources/uriAliasDoesntBeginWithForwardSlash.cfc

```diff
@@ -0,0 +1,7 @@
+<cfcomponent extends="taffy.core.resource" taffy:uri="/uriWithForwardSlash{,},uriAliasWithoutFowardSlash">
```

Comment:
To make sure that if the bracket escaping functionnality breaks in splitURIs(), it doesn't break this one too. Could put it in another test case as it's not obvious I'm implicitely testing this too.